webplugin/js/app/zendesk-chat-service.js

```diff
@@ -28,10 +28,29 @@ function ZendeskChatService() {
             return;
         }
         window.console.log("handleUserMessage: ", event);
-        zChat.sendChatMsg(event.message.message, function (err, data) {
-            window.console.log("zChat.sendChatMsg ", err, data)
-        });
-        // TODO look for attachment
+        if(event.message.contentType==KommunicateConstants.MESSAGE_CONTENT_TYPE.DEFAULT){ //if user sends normal message
+            zChat.sendChatMsg(event.message.message, function (err, data) {
+                window.console.log("zChat.sendChatMsg ", err, data)
+            });
+
+        }else{ //if user sends file attachments
+            let {name, thumbnailBlobKey, url, createdAtTime, contentType} = event.message.file;
+
```

Comment:
the else part will get executed for location sharing also 
can you place a check?
Will switch case be better?

Reply:
Yes, used `else if` instead of `else`.